rust/main/utils/run-locally/src/main.rs

```diff
@@ -539,6 +540,9 @@ fn main() -> ExitCode {
 
         sleep(Duration::from_secs(5));
     }
+    // test retry request
+    let resp = server::run_retry_request().expect("Failed to process retry request");
```

Comment:
if `SUBMITTER_QUEUE_COUNT` was ever inaccurate, would we expect this to fail?

happy to ship either way

Reply:
no, I think it would just be a bit slow to receive a response

Reply:
> if SUBMITTER_QUEUE_COUNT was ever inaccurate, would we expect this to fail?

this check relies on there being unprocessable messages in queues (`ZERO_MERKLE_INSERTION_KATHY_MESSAGES` + `SOL_MESSAGES_WITH_NON_MATCHING_IGP`)

so the assumption is:
- we'll keep having these unprocessable messages
- they'll either be in the prepare or submit queues
----
I would however like this to fail if things get too slow (for whatever reason, including more queues), so in `run-locally/src/server.rs` it'd be great to add a `.timeout(100ms)` when building `client.post(url).json(&body).send()`